src/main/scala/org/camunda/dmn/Audit.scala

```diff
@@ -27,11 +27,7 @@ object Audit {
     def onFailure(log: AuditLog)
   }
 
-  case class AuditLog(dmn: ParsedDmn, entries: List[AuditLogEntry]) {
-
-    val rootEntry = entries.last
```

Comment:
❓ Why was `rootEntry` the last entry of the list? I realise it's gone and I don't really need to worry about it, but it seems odd that the root is the tail of the list instead of head.

Reply:
Good question. The entries are in the order of the evaluation. Since all required decisions are evaluated before the root/called decision, the root decision is at the end. 

As an alternative, we could return the log as an evaluation tree that reflects the requirements as leaves. The structure and logic would be more complex but possible. 

The current structure seems to work. But we could change it if we see any needs.